hr_timesheet_portal/security/hr_timesheet_portal_security.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2021 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="group_hr_timesheet_portal" model="res.groups">
+    <field name="name">Editable timesheets</field>
+    <field name="category_id" ref="base.module_category_portal"/>
```

Comment:
According to Travis test failure, `base.module_category_portal` doesnt exist and I also can find it anywhere. Maybe `base.module_category_website`?

Reply:
yes, looking better now: https://travis-ci.com/github/OCA/timesheet/jobs/486713259#L1797

I had this xmlid in my dev db because it's migrated from an earlier version